locales/en-US/App.json

```diff
@@ -232,5 +232,6 @@
   "into": "into",
   "out from": "out from",
   "Transaction details": "Transaction details",
-  "Copy address": "Copy address"
+  "Copy address": "Copy address",
+  "Moved": "Moved"
```

Comment:
We need this in French

Reply:
I'll add those to #371 is it's fine by you, just to avoid going through merge conflicts.